### DISCUSSION TITLE
Feature Suggestion Hide/unhide

### DISCUSSION THREAD

**Discussion by lordlony1** [bcssov/IronyModManager#336] [Ideas]
Was just wondering if it would be at all possible for a hide unhide feature to be added.

Find myself constantly merging small mods together patches and the like but find it extremally frustrating trying to search through the many different mods installed, especially after disabling them. In some cases I forget they have been merged and still find myself adding them back in only to have longer wait and confusion over why there are then so many conflicts.

I'm unclear how feasible it would be to add this function, I was thinking along the lines of a hide/show button next to search bar to display hidden mods. After a merge the program could then perhaps offer to autohide of any or all merged mods. 
At basic mods could be hidden from view with right click selection similar to the lock unlock options. The option to then view them again with either a hidden mods page or be redisplayed in the installed mods section, ideally distinguishable from other installed mods. 

Believe I was thinking along the routs [Mod Organizer (MO)] (https://www.modorganizer.org/)

My coding knowledge is limited so if this is not possible or even not something you want to introduce I understand, appreciate your time and will love and continue to use regardless.

Thank you.

**Comment by bcssov** [comment]
I'll consider it. I'll leave this discussion open for future ideas and inspirations. If these ever make it into the program I cannot guarantee in which form they'll be in. The way you use Irony is certainly not how I intended merges to be used. Also, I've never used Mod Organizer.

**Reply by lordlony1** [reply]
Another suggestion would to be able to sort between mods that are achievement compatible or not  😁

**Reply by bcssov** [reply]
One can already filter by this status.

**Reply by lordlony1** [reply]
> One can already filter by this status.

I did not know there was [filtering syntax](https://github.com/bcssov/IronyModManager/wiki/Installed-Mods) apologies.
